runtime/src/lib.rs

```diff
@@ -277,7 +291,14 @@ pub type RateLimiter<K, C> = governor::RateLimiter<
 ///
 /// It is necessary to mock time to provide deterministic execution
 /// of arbitrary tasks.
-pub trait Clock: Clone + Send + Sync + 'static {
+pub trait Clock:
```

Comment:
Alternatively, we could create a `RateLimitedClock` or `GovernedClock` but I think keeping them closely related is more than OK for now (given we almost always use RL with the clock).